Scripts/signin.js

```diff
@@ -1,48 +1,57 @@
+const emailInput = document.querySelector('#emailInput');
+const pwInput = document.querySelector('#pwInput');
 const confirmBtn = document.querySelector('#confirmBtn');
 
 // 에러 메시지 출력 장소
-const emailErrorSection = loginEmailInput.parentElement.lastElementChild;
-const passwordErrorSection = loginPasswordInput.parentElement.parentElement.lastElementChild;
+const emailErrTextSection = document.querySelector('#emailErr')
+const pwErrTextSection = document.querySelector('#pwErr');
 
 
 // 이메일 리스너
-loginEmailInput.addEventListener('input' , () => inputFocus(emailErrorSection));
-loginEmailInput.addEventListener('focus' , () => inputFocus(emailErrorSection));
-loginEmailInput.addEventListener('blur' , () => inputBlur(errorType.email, emailErrorSection));
+emailInput.addEventListener('input' , (e) => focusToInput(e, emailErrTextSection));
+emailInput.addEventListener('focus' , (e) => focusToInput(e, emailErrTextSection));
+emailInput.addEventListener('blur' , (e) => blurFromInput(e, emailErrTextSection, ERROR_TYPE.EMAIL_SECTION_BLANK));
 
 
 // 패스워드 리스너
-loginPasswordInput.addEventListener('input' , () => inputFocus(passwordErrorSection));
-loginPasswordInput.addEventListener('focus' , () => inputFocus(passwordErrorSection));
-loginPasswordInput.addEventListener('blur' , () => inputBlur(errorType.password, passwordErrorSection));
+pwInput.addEventListener('input' , (e) => focusToInput(e, pwErrTextSection));
+pwInput.addEventListener('focus' , (e) => focusToInput(e, pwErrTextSection));
+pwInput.addEventListener('blur' , (e) => blurFromInput(e, pwErrTextSection, ERROR_TYPE.PW_SECTION_BLANK));
 
 
 // 로그인 정보 제출
-const submitLoginData = function (event)  {
-   event.preventDefault();
+const signinDataSubmit = async function (e)  {
+   e.preventDefault();
    
    //입력받은 값
-   const emailValue = loginEmailInput.value;
-   const passwordValue = loginPasswordInput.value;
-   
-   // 계정정보 유효 확인
-   const isAccountSigned = loginCheck(emailValue, passwordValue);
+   const emailValue = emailInput.value;
+   const passwordValue = pwInput.value;
+   const triedLoginAccountData = {
+      email : emailValue,
+      password : passwordValue
+   }
 
-   if (!isAccountSigned) {
-      printError(loginEmailInput, emailErrorSection, errorType.mismatchEmail);
-      printError(loginPasswordInput, passwordErrorSection, errorType.mismatchPassword);
-   } else {
+   try {
+      const signinAttemptResponse = await isThisLoginWasSuccessful(triedLoginAccountData)
+      if (!signinAttemptResponse) {
+         throw new Error()
+      }
       window.location.href = './folder.html';
+   } catch {
+      errPrint(emailInput, emailErrTextSection, ERROR_TYPE.EMAIL_MISMATCH_WITH_ACCOUNT);
+      errPrint(pwInput, pwErrTextSection, ERROR_TYPE.PW_MISMATCH_WITH_ACCOUNT);
    }
 }
 
-confirmBtn.addEventListener('click' , submitLoginData);
\ No newline at end of file
+confirmBtn.addEventListener('click' , (e) => signinDataSubmit(e));
```

Comment:
다른 매개변수를 넘겨주시는 것이 아니라면 아래처럼도 가능합니다.

```suggestion
confirmBtn.addEventListener('click' , signinDataSubmit);
```

Reply:
알겠습니다! 감사합니다!